bats/tests/helpers/load.bash

```diff
@@ -86,6 +89,8 @@ teardown_file() {
         run rdctl shutdown
     fi
 
+    teardown_ramdisk
```

Comment:
You didn't respond to this question:

> Since you call `setup_ramdisk` after `call_local_function` because the latter may disable it, shouldn't `teardown_ramdisk` also be called after, for symmetry, if nothing else?

Contrived example: if you `call_local_function` first, then it still has access to the content of the ramdisk; if you call it last, then the data is already gone, limiting what you can do in `local_teardown_file`.